src/main/java/de/tum/in/www1/artemis/service/notifications/MailService.java

```diff
@@ -324,6 +324,8 @@ private String createContentForNotificationEmailByType(NotificationType notifica
                     .process("mail/notification/tutorialGroupBasicEmail", context);
             case TUTORIAL_GROUP_DELETED -> templateEngine.process("mail/notification/tutorialGroupDeletedEmail", context);
             case TUTORIAL_GROUP_UPDATED -> templateEngine.process("mail/notification/tutorialGroupUpdatedEmail", context);
+            case DATA_EXPORT_CREATED -> templateEngine.process("mail/notification/dataExportCreatedEmail", context);
+            case DATA_EXPORT_FAILED -> templateEngine.process("mail/notification/dataExportFailedEmail.html", context);
```

Comment:
I don't really know how the template Engine works, but in all other cases the `.html` is missing. Is this maybe a bug here?

Reply:
i added it by accident, removed now.